regions/_geometry/core.pxd

```diff
@@ -1,4 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+# cython: language_level=3
```

Comment:
Also fixed some Cython warnings I saw when installing the package from source. Follow up of #270